File: xcode/JwaooToy/JwaooToy/JwaooToyParser.m
//
//  JwaooToyParser.m
//  JwaooToy
//
//  Created by 曹福昂 on 16/7/14.
//  Copyright © 2016年 曹福昂. All rights reserved.
//

#import "JwaooToyParser.h"

@implementation JwaooToyParser

-(void)putSensorData:(JwaooToySensor *)sensor {
    [super putCapacityValue:sensor.getCapacitys];
}

@end

File: android/app/CavanAndroid/src/com/jwaoo/android/JwaooToyParser.java
package com.jwaoo.android;


@SuppressWarnings("serial")
public class JwaooToyParser extends JwaooDepthDecoder {

	public JwaooToyParser(double fuzz) {
		super(fuzz);
	}

	public void putData(JwaooToySensor sensor) {
		putCapacityValue(sensor.getCapacitys());
	}
}